Sanya_form.HTML

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Intro to HTML and Javascript</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+   
+    <form>
+      <b><label>First Name:</label></b>
+      <input 
+            id="first_name" 
+            type="text" 
+            placeholder="Name"
+            maxlength="10" 
+            minlength="10"/>
+        
+        <br/>
+        
+      <b><label>Last Name:</label></b>
+      <input id="last_name" type="text" placeholder="Name" />
+        
+        <br/>
+
+      <b><label>Date_of_Birth:</label></b>
```

Comment:
No need for the underscore in the label text. The user of your website will be able to see the label, so use plain text:  "Date of Birth"